src/Agent.Listener/DistributedTask.Pipelines/Yaml/TypeConverters/ConverterUtil.general.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using YamlDotNet.Core;
+using YamlDotNet.Core.Events;
+using YamlDotNet.Serialization;
+
+namespace Microsoft.TeamFoundation.DistributedTask.Orchestration.Server.Pipelines.Yaml.TypeConverters
+{
+    internal static partial class ConverterUtil
```

Comment:
All of these static util functions were previously on the YamlTypeConverter base class. I deleted the base class and converted to static util functions, since we don't really need inheritance for anything.

However, there are so many util functions so I split into partial files as an attempt to logically group and improve readability.